html/js/display-product.js

```diff
@@ -35,22 +35,22 @@ class RobotoffAsker extends HTMLElement {
         .hidden { display: none; }
       </style>
       <div class="row hidden zoomrow" id="zoomrow">
-        <div class="medium-12 columns">
+        <div class="small-12 columns">
           <img src="" id="zoom" data-zoom-in-src="" data-zoom-out-src="">
         </div>
       </div>
       <div class="row">
-        <div class="medium-2 columns">
+        <div class="small-2 columns hidden">
```

Comment:
Wouldn't this *always* hide the thumbnail? I don't see any change removing the hidden class.

Reply:
No thumbnail: 
![image](https://user-images.githubusercontent.com/87124/58907632-f028ac00-870e-11e9-93bb-b8d7b891d1ac.png)

Reply:
```suggestion
        <div class="small-2 columns">
```

Reply:
In fact I do want to always hide the thumbnail, I don't see the point of showing a thumbnail here. I added the hidden class in case we want to put it back at some time for some reason.

![image](https://user-images.githubusercontent.com/8158668/58912556-9b8b2e00-871a-11e9-86b3-be3ea4161d21.png)

Reply:
I forgot about the zoom when clicking on the thumbnail, I guess the thumbnail can be useful in some cases, if the selected picture is different.

Reply:
Yes, the thumbnail was added to enable users to see the source of Robotoff's suggestion. 🙂 Even the small image can be too small to identify the text, which is why I added the optional (rudimentary) zoom levels.